docs/public/deploying-java-app-with-database.adoc

```diff
@@ -0,0 +1,432 @@
+= Binding a Java microservices JPA application to an in-cluster Operator-Managed PostgreSQL Database on minikube
+
+This scenario illustrates binding an `odo` managed Java MicroServices JPA application to an in-cluster Operator managed PostgreSQL Database in the minikube environment.
+
+== Installation and Configuration of the minikube environment
+[NOTE]
+====
+The guide only works with minikube configured with kubernetes 1.19.x or lower. `odo link` cannot link services successfully in a Kubernetes 1.20.x or newer environment.
+====
+
+It is recommended that users of this guide obtain a suitable system for running minikube with kubernetes. In practice this should be a 4 core system minimum. Before proceeding to the sample application, please follow the instructions for establishing a minikube environment:
+
+=== Installing and starting docker 
```

Comment:
This might need to be removed if we decide to go ahead with my idea of starting minikube as a VM instead of using docker driver.

Reply:
@dharmit right, I forgot to commit the changes in that Java doc. -_- Apologies, fixed.